drivers/media/platform/raspberrypi/rp1_cfe/dphy.c

```diff
@@ -128,6 +129,19 @@ static void dphy_set_hsfreqrange(struct dphy_data *dphy, uint32_t mbps)
 			 hsfreqrange_table[i][1] << 1);
 }
 
+static void dphy_set_manual_clock_term(struct dphy_data *dphy)
+{
+	/* See page 162 of the dphy databook */
+	const u8 tst = (1 << 0) + /* HS RX power on */
+		       (1 << 1) + /* Bypass HS RX power on */
+		       (1 << 2) + /* HS RX cal enable */
+		       (0 << 3) + /* HS RX cal control */
+		       (1 << 4) + /* Term EN */
+		       (1 << 5) + /* Term EN control */
```

Comment:
Typo here. You want the line ending with `;`, not `+`
It's why the kernel build has failed.

Reply:
Hmmm, why did this pass my compile test, or did it??

Reply:
Warnings are errors is only enabled for CI builds, not by default.

I saw the CI error and looked more closely. My build had the warning, and it took a couple of seconds to twig as to why. The compiler was absolutely correct that tst was used uninitialised.